src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -589,6 +603,7 @@ export function DashboardTopNav({
 
   return (
     <>
+      {dashboardState.viewMode === ViewMode.VIEW && <DashboardViewTour />}
```

Comment:
I had to put the view mode tour in `dashboard_top_nav.tsx` so that I could have direct access to the `onClick` action of switching to edit mode - that way, either by directly dismissing it or by switching from view mode **to** edit mode, the view mode tour will be dismissed permanently.

Since it would have made the code much cleaner, I originally tried to do this by injecting the view mode tour alongside the edit mode tour in the Dashboard viewport and listening for **changes** to the view mode to trigger the ending of the view mode tour - unfortunately this happened **too late** and I would get the following error: 

![image](https://user-images.githubusercontent.com/8698078/167885775-e9bb1be8-b753-49aa-8528-ba7323c034c8.png)

which was caused by the `EuiTourStep` being anchored to the edit button, which is dynamically hidden and replaced with the `View mode` button when switching from view mode to edit mode. 

Very much interested in seeing if there is a better way of handling this 👍